### ISSUE TITLE
Unfair treaty of versailles for Bulgaria in Entente

### ISSUE THREAD

**Issue by tihhoni** [Wolferos/Hearts-of-Iron-IV-The-Great-War#43]
Hey, just had a fun run as Bulgaria until the peace treaty fired
Some history:
- Ottomans win vs Italy (instant)
- Won Balkan league vs Ottomans
- Went on Demand more land -> Greater Bulgaria vs Anti-Bulgarian league
- Almost lost it, but once ww1 start, Serbia opt out of my war (others stayed)
- After that some weird stuff happened, Ottomans basically just left all fronts
my guess is that army got kicked out from Serbia to Montenegro, i have went through whole Turkey territory and have seen only ~7 divisions (but should have been 100+)
Hejaz was the major! :D the rest did not count as one
- I capitulated Greece, Romania, Turkey and Hejaz

In the peace treaty i puppeted Syria, Lebanon, Palestine, Jordan, Iraq, Livia, Yemen, Asir
(Gave Hejaz to Njed)
Full annex Romania, Greece and Turkey

- After that was just observing WW1 and lend leasing equipment to central powers
- At 1920 decided to join on Entente side and after few months - Treaty of Versailles

My fun ended when i saw the result:
- Serbia took East Macedonia and Montenegro from me
- Romania back to live and takes everything (even Dobrudja)
- Greece back to live in thrace
- My Asir puppet annexed by Njed
- France annexes my puppets Syria and Lebanon
- UK annexes my puppets Irag, Jordan, Palestine

This is how Entente treats me 🗡 

Probably the script is missing some flags to check or Anti-Bulgarian league should white peace Bulgaria

Save game attached (just once Peace treaty happen):
[unfair.hoi4.zip](https://github.com/Wolferos/Hearts-of-Iron-IV-The-Great-War/files/752988/unfair.hoi4.zip)

**Comment by Wolferos** [comment]
Hey @tihhoni 

Thanks for your AAR.

- The Italo-Turkish War will be fixed for the next Patch.
- We still have to work on Treaties and find a good comprise for each nation that is at war. Though we are limited by what the engine let us do. 
For example, we can get the warscore participation for a specific country. We can't get a list of all the countries that are in a specific faction, we have to check one by one.

When WW1 started, was it because of the Austria-Hungarian Intervention? You were still at war with Greece, Romania and the Ottoman Empire?

**Comment by tihhoni** [comment]
Yes WW1 started with "Instability on the Balkans" event, i was hoping it would peace me out as well but it did not. I stayed at war vs Anti-Bulgarian league (Montenegro-Greece-Romania-Turkey-Hejaz)

I guess you are never meant to win this war. I had a brilliant plan - capitulate Romania fast and push to Constantinople strait, but have failed to do so. Romania reinforced Bucharest fast and i could not get to Constantinople either (I am pretty sure i can do that with a few save - load try outs).

It turned out to be a trench war for a year, until i run out of equipment and manpower. I became desperate :) started requesting lend lease from Austria-Hungary, Germany, Russia, but those "did not have a valid trade path" (even though i have connection to Russia through Black sea)

Then i dismantled my "Greater Bulgaria" faction in the hopes i can join Entente. Which also failed, France loves me not.

I guess the AI Bulgaria can never win this war, but human can. So maybe having white peace for Bulgaria winning / no progress for the long time would be good. And also i suppose WW1 should stop the Second Balkan war (It stops the first one right?)
